specification/eventgrid/data-plane/Microsoft.ApiManagement/stable/2018-01-01/APIManagement.json

```diff
@@ -156,6 +156,116 @@
+          "type": "string"
+        }
+      }
+    },
+    "ApiManagementSelfHostedGatewayHostnameConfigurationDeletedEventData": {
+      "description": "Schema of the Data property of an EventGridEvent for a Microsoft.ApiManagement.SelfHostedGateway.HostnameConfiguration.Deleted event.",
+      "type": "object",
+      "properties": {
+        "resourceUri": {
+          "description": "The fully qualified ID of the resource that the compliance state change is for, including the resource name and resource type. Uses the format, `/subscriptions/<SubscriptionID>/resourceGroups/<ResourceGroup>/Microsoft.ApiManagement/service/<ServiceName>/gateways/<GatewayName>/hostnameConfigurations/<ResourceName>`",
+          "type": "string"
+        }
+      }
+    },
+    "ApiManagementSelfHostedGatewayCertificateAuthorityCreatedEventData": {
+      "description": "Schema of the Data property of an EventGridEvent for a Microsoft.ApiManagement.SelfHostedGateway.CertificateAuthority.Created event.",
+      "type": "object",
+      "properties": {
+        "resourceUri": {
+          "description": "The fully qualified ID of the resource that the compliance state change is for, including the resource name and resource type. Uses the format, `/subscriptions/<SubscriptionID>/resourceGroups/<ResourceGroup>/Microsoft.ApiManagement/service/<ServiceName>/gateways/<GatewayName>/certificateAuthorities/<ResourceName>`",
+          "type": "string"
+        }
+      }
+    },
+    "ApiManagementSelfHostedGatewayCertificateAuthorityUpdatedEventData": {
+      "description": "Schema of the Data property of an EventGridEvent for a Microsoft.ApiManagement.SelfHostedGateway.CertificateAuthority.Updated event.",
+      "type": "object",
+      "properties": {
+        "resourceUri": {
+          "description": "The fully qualified ID of the resource that the compliance state change is for, including the resource name and resource type. Uses the format, `/subscriptions/<SubscriptionID>/resourceGroups/<ResourceGroup>/Microsoft.ApiManagement/service/<ServiceName>/gateways/<GatewayName>/certificateAuthorities/<ResourceName>`",
+          "type": "string"
+        }
+      }
+    },
+    "ApiManagementSelfHostedGatewayCertificateAuthorityDeletedEventData": {
+      "description": "Schema of the Data property of an EventGridEvent for a Microsoft.ApiManagement.SelfHostedGateway.CertificateAuthority.Deleted event.",
+      "type": "object",
+      "properties": {
+        "resourceUri": {
+          "description": "The fully qualified ID of the resource that the compliance state change is for, including the resource name and resource type. Uses the format, `/subscriptions/<SubscriptionID>/resourceGroups/<ResourceGroup>/Microsoft.ApiManagement/service/<ServiceName>/gateways/<GatewayName>/certificateAuthorities/<ResourceName>`",
+          "type": "string"
+        }
+      }
+    },
+    "ApiManagementSelfHostedGatewayAPIAddedEventData": {
```

Comment:
The `API` should be cased as `Api`

Reply:
I presume this comment is just for this entry and our effective event type is something we are allowed to define? Because that's `Microsoft.ApiManagement.SelfHostedGateway.API.Added`

Reply:
Done